### DISCUSSION TITLE
請問還會再更新嗎?

### DISCUSSION THREAD

**Discussion by JieuTang** [Chinlinlee/NTUNHS-Assistant#5] [Q&A]
請問還會再更新嗎?
![20220608_042538072_iOS](https://user-images.githubusercontent.com/41420510/174489602-00de7266-84d2-4458-b2e1-b912a64584a6.jpg)

**Comment by Chinlinlee** [comment]
由於小弟我已經不是這間學校的學生了，所以比較無法進行更新與測試。
並且此專案開源的目的就是期望北護的學生能夠持續改進及維護此專案。
